project/overview.md

```diff
@@ -0,0 +1,37 @@
+<!-- vim: ft=markdown colorcolumn=72
+-->
+# Project Overview
+
+The spire/helm-chart effort intends to deliver a set of helm charts to
+support SPIRE deployments in Kubernetes to the widest possible audience of
+users.
+
+## Mission and Scope
+The mission of this effort is to provide a stable upstream repository
+of Helm Charts sufficient for any organization or individual to use.
+
+### What problem does this project address?
+Prior to this effort, no single authoritative repository for SPIRE Helm
+Charts existed, imposing an additional burden on SPIRE adoption as each
+organization or individual had to develop their own Kubernetes
+deployment packaging.
+
+### What is the goal of this project?
+The goal of this project is to be the primary means of deployment for
+SPIRE into Kubernetes clusters, providing a flexible solution that can
+be used in a wide number of different scenarios.
+
+### What is the scope of this project?
+The scope includes user documentation, chart creation, chart delivery,
+chart testing, and chart maintenance of a set of Helm Charts sufficient
+for deploying SPIRE in any SPIRE documented configuration, including
+direction on how to extend the Charts to integrate non-SPIRE plugins.
+
+### What development methodology is used?
+An Open Source Development methodology is used, where issues are tracked
+within the primary github repository.
+
+## Status
+The project is currently in a pre-release status.  While the standalone
+deployment is close to release quality, the tiered deployment is not.
```

Comment:
We might want to more explicitly define what `tiered` deployment means (nested and federated).